### PR TITLE
Update Podspec to use RAC 4.2.1

### PR DESCRIPTION
**Pull request by minuscorp**
Maybe a version bump and/or a release would be better than applying a
defined commit number.

### CODE REVIEW

**Comment by minuscorp** [comment]
I don't know why this is not linting on Travis, as it goes fine in local, I think creating a tag and modifying the target tag for the podspec should fix it.

File: Rex.podspec [hunk 1]
```diff
@@ -18,8 +18,8 @@ Pod::Spec.new do |s|
   s.watchos.deployment_target = '2.0'
   s.tvos.deployment_target = '9.0'
 
-  s.source       = { :git => 'https://github.com/neilpa/Rex.git', :tag => s.version }
-  s.dependency 'ReactiveCocoa', '~> 4.1'
+  s.source       = { :git => 'https://github.com/neilpa/Rex.git', :commit => '265e8e9c9503bc917ee5af5252590ab38eda4bff' }
```
Comment (bjarkehs) [inline Rex.podspec]: No need to change this line. We should simply have @neilpa cut a new version. It is a little impractical we are so heavily tied to to the RAC version.

It fails linting due to:

```
-> Rex (0.11.0)
    - WARN  | xcodebuild:  ReactiveCocoa/ReactiveCocoa/Objective-C/RACCompoundDisposable.m:85:12: warning: unused variable 'result' [-Wunused-variable]
    - WARN  | xcodebuild:  ReactiveCocoa/ReactiveCocoa/Objective-C/RACCompoundDisposable.m:132:12: warning: unused variable 'result' [-Wunused-variable]
    - WARN  | xcodebuild:  ReactiveCocoa/ReactiveCocoa/Objective-C/RACSerialDisposable.m:63:12: warning: unused variable 'result' [-Wunused-variable]
    - WARN  | xcodebuild:  ReactiveCocoa/ReactiveCocoa/Objective-C/RACSerialDisposable.m:79:12: warning: unused variable 'result' [-Wunused-variable]
```

I guess there is not really any other way around this than to simply ignore the failed builds so far.

**Comment by bjarkehs** [comment]
A dirty fix for us CocoaPods lovers could be to make a 0.10.0a or similar that's tied to a specific commit. But I would prefer going through official releases :)

**Comment by minuscorp** [comment]
I'm really waiting for someone to make a new tag in the repo so I can change the podspec, no intention in merge this as it is right now because I understand how dirty my initial solution is ;)

File: Rex.podspec (hunk 1 above)
Comment (minuscorp) [inline Rex.podspec]: I didn't actually knew that warnings made linting to fail. I might have expected that the CI would execute the `--allow-warnings` by default.

**Comment by neilpa** [comment]
I just added a `0.11.0-beta.1` tag but don't have time currently to cut an official release w/ notes.

File: Rex.podspec [hunk 2]
```diff
@@ -18,7 +18,7 @@ Pod::Spec.new do |s|
   s.watchos.deployment_target = '2.0'
   s.tvos.deployment_target = '9.0'
 
-  s.source       = { :git => 'https://github.com/neilpa/Rex.git', :commit => '265e8e9c9503bc917ee5af5252590ab38eda4bff' }
+  s.source       = { :git => 'https://github.com/neilpa/Rex.git', :tag => '0.11.0-beta.1' }
```
Comment (bjarkehs) [inline Rex.podspec]:
```
:tag => s.version
```

Comment (minuscorp) [inline Rex.podspec]: My bad :sweat_smile:

File: Rex.podspec [hunk 3]
```diff
@@ -19,7 +19,7 @@ Pod::Spec.new do |s|
   s.tvos.deployment_target = '9.0'
 
   s.source       = { :git => 'https://github.com/neilpa/Rex.git', :tag => s.version }
```
Comment (neilpa) [inline Rex.podspec]: You should update this to `RACCommunity/Rex` now that I've transferred the repo

Comment (neilpa) [inline Rex.podspec]: And the `homepage` field above too

Comment (minuscorp) [inline Rex.podspec]: Of course! Didn't know about it, sorry.

Comment (bjarkehs) [inline Rex.podspec]: @neilpa what about author?

Comment (neilpa) [inline Rex.podspec]: Technically I'm more or less still the author. We could add others since _I think_ that supports a list of authors.

Comment (bjarkehs) [inline Rex.podspec]: I wasn't trying to discredit you, just wondering if it should be updated too :)

Comment (neilpa) [inline Rex.podspec]: I know you weren't. I'm just not sure what we would put there instead.

**Comment by dmcrodrigues** [comment]
Superseded by #141.

**Pull request closed without merging**